ros2-library/src/main/java/us/ihmc/ros2/ROS2Node.java

```diff
@@ -28,24 +27,15 @@ public class ROS2Node
 {
    static int DEFAULT_QUEUE_SIZE = 10;
 
-   public static final String DEFAULT_NAMESPACE = "/us/ihmc";
-
    private Domain domain;
    private Participant participant;
 
    private final String nodeName;
    private final String namespace;
 
-   /**
-    * Create a ROS2Node
-    * @param domain                 DDS domain to use. Use DomainFactory.getDomain(implementation)
-    * @param name                   Name of the ROS 2 node
-    * @param namespace              Namespace of the ROS 2 node
-    * @param attributes             ParticipantAttributes for the domain
-    */
-   public ROS2Node(Domain domain, String name, String namespace, ParticipantProfile attributes)
+   protected ROS2Node(String name, String namespace, ParticipantProfile attributes)
    {
-      this.domain = domain;
+      this.domain = DomainFactory.getDomain();
```

Comment:
I think I would print here which domain we ended up going with. It's final at this point, right?